pandunia-lekse.csv

```diff
@@ -29,6 +29,8 @@ aga|||fas:urd:اگر‎‎ (agar), hin:अगर (agar), tur:eğer, + yue:假 (g
 agrese|||eng:deu:fra:agression, pol:agresja, rus:агрессия (agressiya), spa:agresión, por:agressão, may:agresif|attack (assault, aggression)|agression (attaque)|ataque|ataque|нападение (атака)||攻击||||आक्रमण||menarik||saldırı|atako (agreso)|hyökkäys (aggressio)|atak (agresja, napaść)
 agresi||||aggressive||agresivo|||||||||||||agresema|hyökkäävä (aggressiivinen)|agresywny
 aha|||rus:ага (aha), kor: 아하 (aha), eng: uh-huh, pol:aha|understand (realize)|comprendre (réaliser)|entender (comprender)|||||||||||||kompreni|oivaltaa (tajuta)|rozumieć, pojmować
+ahota|||hin:आखेट (ākheṭ), ben:খোজা (khōjā), eng:hunt, rus:охота (ohota), eng:hunt|hunt|chasser|cazar|caçar|охотиться||打猎|狩る||||||||||
```

Comment:
me no suku yi cuze. asli lekse no forma veri grupe.
me wana uza sami lekse pa "hunting (try to catch game)" e "fishing (try to catch fish)". me fikra arabi lekse "s-y-d" abla funcu. vida https://en.wiktionary.org/wiki/%D8%B5%D8%A7%D8%AF#Arabic
hindi, urdi e farsi uza lekse "said" o "seyd".

Reply:
vero? me no abla eureka "said" o "seyd" sa mi hindi leksoliste o sa gugol tarjer. aga sahi, me suku "saida" visa "ahota".